conjur/constants.py

```diff
@@ -11,4 +11,4 @@
 
 DEFAULT_CONFIG_FILE = os.path.expanduser(os.path.join('~', '.conjurrc'))
 DEFAULT_NETRC_FILE = os.path.expanduser(os.path.join('~', '.netrc'))
-CERTIFICATE_FILENAME = "conjur-server.pem"
+FULL_CERTIFICATE_PATH=os.path.expanduser(os.path.join('~', "conjur-server.pem"))
```

Comment:
Why did you name it differently than the others?
It is a default path, and it's a file, so DEFAULT_CERTIFICATE_FILE seems correct to me.